Cargo.toml

```diff
@@ -42,13 +42,12 @@ bevy = { version = "0.14", default-features = false, features = [
 ] }
 
 [features]
-default = ["client", "server", "renet_serde", "renet_transport"]
+default = ["client", "server", "renet_netcode"]
 server = ["bevy_replicon/server"]
 client = ["bevy_replicon/client"]
 
 # Re-exports of renet features
-renet_serde = ["bevy_renet/serde"]
-renet_transport = ["bevy_renet/transport"]
+renet_netcode = ["bevy_renet/netcode"]
```

Comment:
We need to re-export the steam feature.

Reply:
Added, I follow `bevy_renet` and kept it as an optional feature. Would that be the correct way to go about it?

Reply:
Yes, I would do the same!